api_tests/files/views/test_file_detail.py

```diff
@@ -71,6 +71,7 @@ def test_get_file(self):
         assert_equal(attributes['last_touched'], None)
         assert_equal(attributes['provider'], self.file.provider)
         assert_equal(attributes['size'], self.file.versions[-1].size)
+        assert_equal(attributes['current_version'], len(self.file.history))
```

Comment:
I left this base test as is.

Added another test that went through a for loop creating versions of files and then requesting that file, checking the version was the latest version.